docs/span_attribute_schema/v1.md

```diff
@@ -226,7 +226,7 @@ span.kind | `server`
 ### Span properties
 Name | Required |
 ---------|----------------|
-Name | `grpc.request`
+Name | `grpc.request`; `grpc.server.request`
```

Comment:
Not for this PR (and maybe overkill anyway), but maybe it would be good to differentiate the use cases in 2 different tables (ie a table for client grpc client request, and one for grpc server requests). Instead of having the 2 in the same table. 
But that super nit picky.

Reply:
100% agree. Maybe I could do that in my subsequent gRPC client PR